SimpleWAWS/Global.asax.cs

```diff
@@ -132,16 +137,22 @@ protected void Application_BeginRequest(Object sender, EventArgs e)
             ExperimentManager.AssignExperiment(context);
             GlobalizationManager.SetCurrentCulture(context);
 
-            //if (context.Request.Cookies[Constants.TiPCookie] == null &&
-            //    context.Request.QueryString[Constants.TiPCookie] != null)
-            //{
-            //    context.Response.Cookies.Add(new HttpCookie(Constants.TiPCookie, context.Request.QueryString[AuthConstants.TiPCookie]) { Path = "/" });
-            //}
+            context.Response.Headers["Access-Control-Expose-Headers"] = "LoginUrl";
```

Comment:
From what I read I dont think it allows for `Access-Control-Expose-Headers`.  All it allows for is `Access-Control-Allow-Origin` . Can you point me to how we can update the cors json section to enable this ?